addons/udes_delivery_control/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_udes_picking_extras,access_udes_picking_extras,model_udes_picking_extras,base.group_user,1,0,0,0
```

Comment:
We need to review the security permissions, because at present only admin user would be able to create the delivery control records at the moment.

Reply:
I gave the model default stock_picking permission. We can change in future but it will stop odoo from giving us warning message :P

Reply:
As discussed added additional permissions

Reply:
Stock Pickings in Odoo's stock module also have these permissions for stock user and portal user:

```
access_stock_picking_user,stock.picking user,model_stock_picking,stock.group_stock_user,1,1,1,1
access_stock_picking_portal,stock.picking,stock.model_stock_picking,base.group_portal,1,0,0,0
```

I think stock user should be copied across for Picking Extras, not sure about portal user.

Reply:
Removed other permissions and replace group_user with stock.group_stock_user